src/NServiceBus.Metrics/MetricsOptions.cs

```diff
@@ -19,69 +11,36 @@ public class MetricsOptions
         /// Enables sending periodic updates of metric data to ServiceControl
         /// </summary>
         /// <param name="serviceControlMetricsAddress">The transport address of the ServiceControl instance</param>
-        /// <param name="interval">How frequently metric data is sent to ServiceControl</param>
+        /// <param name="interval">Interval between consecutive reports</param>
         [Obsolete("Not for public use.")]
         public void SendMetricDataToServiceControl(string serviceControlMetricsAddress, TimeSpan interval)
         {
             Guard.AgainstNullAndEmpty(nameof(serviceControlMetricsAddress), serviceControlMetricsAddress);
             Guard.AgainstNegativeAndZero(nameof(interval), interval);
 
-            reportInstallers.Add((builder, config) => config.WithReport(
-                new NServiceBusMetricReport(builder.Build<IDispatchMessages>(), serviceControlMetricsAddress, builder.Build<HostInformation>()),
-                interval
-            ));
+            ServiceControlMetricsAddress = serviceControlMetricsAddress;
+            ReportingInterval = interval;
         }
 
         /// <summary>
-        /// Enables sending metric data to the trace log
+        /// Enables registering observers to available probes.
```

Comment:
Doesn't really _enable_ the registering so much as it actually _does_ register observers.